services/diversity/src/persistence/richness_nos.js

```diff
@@ -0,0 +1,299 @@
+const { areaTypeKeys } = require('../util/appropriate_keys');
+
+module.exports = (
+  db,
+  {
+    richnessNos,
+  },
+  logger,
+) => ({
+  /**
+   * Find the layer for the number of species in the given area of the given group
+   *
+   * @param {String} areaType area type
+   * @param {String | Number} areaId area id
+   * @param {String} group group to select the proper layer, options are: 'total', 'endemic',
+   * 'invasive', 'threatened'.
+   *
+   * @returns {Binary} Image with the geometry
+   */
+  getAreaLayer: (geometry, filename) => (
```

Comment:
Please move this function to the end of the file in order to keep layer persistences at the end as we have in other services.

Reply:
Done!